### PR TITLE
Fixes, cleanup and programs for calibration

### PR DESCRIPTION
**Pull request by jviereck**

### CODE REVIEW

File: demos/nyu_finger_double_calibration_dgh.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        next_time += dt
+
+        ###
+        # Get the latest measurements from the shared memory.
+        head0.read()
+        head1.read()
+
+        ###
+        # Set the P and D gains.
+        P = np.zeros(3)
```
Comment (huaijiangzhu) [inline demos/nyu_finger_double_calibration_dgh.py]: I would set the gains before the loop for better clarity

**Review by huaijiangzhu — COMMENTED** [review]
1. Do we still need ```src/dynamic_graph_manager/dg_main_nyu_finger.cpp```?
2. The naming for Python bindings seems inconsistent and redundant. For example, for DGM, we have ```dynamic_graph_manager_cpp_bindings``` and for HWP we have ```nyu_finger.nyu_finger_hwp_cpp```. If there is no ambiguity, I would simply name them ```dynamic_graph_manager``` and ```nyu_finger.hwp```.

**Comment by MaximilienNaveau** [comment]
>     1. Do we still need `src/dynamic_graph_manager/dg_main_nyu_finger.cpp`?
> 
>     2. The naming for Python bindings seems inconsistent and redundant. For example, for DGM, we have `dynamic_graph_manager_cpp_bindings` and for HWP we have `nyu_finger.nyu_finger_hwp_cpp`. If there is no ambiguity, I would simply name them `dynamic_graph_manager` and `nyu_finger.hwp`.

1. Hi I think it would be preferable to keep the dynamic_graph executable for now.
2. There is ambiguity between the python bindings and the actual python package:
    - The python module is always called with the package name: `import nyu_finger`
    - The python bindings should always contains the package_name as root prefixed by `cpp` or `cpp_bindings`: `import nyu_finger_cpp`

The python bindings library should not be inside the python module.
This helps to differentiate them in the code and it is less verbose.

**Comment by jviereck** [comment]
> The python bindings library should not be inside the python module.
> This helps to differentiate them in the code and it is less verbose.

I agree with Max. For telling that some code comes from c++, we started to use the prefix `_cpp`.  We could drop the `_bindings` in `dynamic_graph_manager_cpp_bindings` if you want.

File: demos/nyu_finger_double_calibration_dgh.py (hunk 1 above)
Comment (jviereck) [reply demos/nyu_finger_double_calibration_dgh.py]: Thanks for pointing this out. I moved the P & D definition outside of the loop now.

**Comment by jviereck** [comment]
> Do we still need src/dynamic_graph_manager/dg_main_nyu_finger.cpp?

No, I think the way to go is to spawn the hardware process from python using the python bindings. This gives us more flexibility. I removed the `src/dynamic_graph_manager/dg_main_nyu_finger.cpp` file.

**Comment by jviereck** [comment]
@MaximilienNaveau @huaijiangzhu can you take another look and if this is good, can we merge it?

**Review by huaijiangzhu — APPROVED** [review]
Looks good to me!

**Pull request merged**